refinery/data_set_manager/views.py

```diff
@@ -311,7 +320,8 @@ def post(self, request, *args, **kwargs):
                     request.user.username,
                     False,
                     response['data']['temp_file_path'],
-                    identity_id=identity_id
+                    identity_id=identity_id,
+                    existing_dataset_uuid=existing_dataset_uuid
```

Comment:
data_set

Reply:
No need for an extra variable: `existing_dataset_uuid=request.GET.get('data_set_uuid')`